platform-sdk/swirlds-merkle/src/test/java/com/swirlds/virtual/merkle/reconnect/VirtualMapJPDBReconnectTest.java

```diff
@@ -362,59 +360,17 @@ void reconnectWithSomeLeavesCleanAndSomeInCache(final int max, final int batches
      * Configure reconnect so that failed reconnect attempts abort very quickly.
      */
     private void configureReconnectToFailQuickly() {
-        ReconnectSettingsFactory.configure(new ReconnectSettings() {
-            @Override
-            public boolean isActive() {
-                return true;
-            }
-
-            @Override
-            public int getReconnectWindowSeconds() {
-                return 0;
-            }
-
-            @Override
-            public double getFallenBehindThreshold() {
-                return 0;
-            }
-
-            @Override
-            public int getAsyncStreamTimeoutMilliseconds() {
-                return 500; // This is important! A low value will cause a failed reconnect to finish more quicly.
-            }
-
-            @Override
-            public int getAsyncOutputStreamFlushMilliseconds() {
-                return 10;
-            }
-
-            @Override
-            public int getAsyncStreamBufferSize() {
-                return 1000;
-            }
-
-            @Override
-            public int getMaxAckDelayMilliseconds() {
-                return 10;
-            }
-
-            @Override
-            public int getMaximumReconnectFailuresBeforeShutdown() {
-                return 0;
-            }
-
-            @Override
-            public Duration getMinimumTimeBetweenReconnects() {
-                return null;
-            }
-        });
-    }
-
-    /**
-     * Reset reconnect settings to their default values.
-     */
-    private void resetReconnectSettings() {
-        ReconnectSettingsFactory.configure(null); // Will cause next call to the settings factory to generate defaults
+        new TestConfigBuilder()
+                .withValue("reconnect.active", "true")
```

Comment:
Should not be done in this PR:

We should create an issue to add the names of all the properties that belong to configs that are part of swirlds-common as constants to the test-fixtures of swirlds-common

Reply:
https://github.com/hashgraph/hedera-services/issues/6281